README.md

```diff
@@ -76,22 +76,42 @@ Custom Slug Generation
 -------
 
 By default Mongoid Slug generates slugs with stringex. If this is not desired you can
-define your own slug generator like this:
-
-```ruby
-class Caption
-  include Mongoid::Document
-  include Mongoid::Slug
-
-  #create a block that takes the current object as an argument
-  #and returns the slug.
-  slug do |cur_object|
-    cur_object.slug_builder.to_url
-  end
-end
-```
+define your own slug generator.
+
+There are two ways to define slug generator
+
+1) Global definition:
+
+   Configure a block in `config/initializers/mongoid_slug.rb` as follows:
+
+   ```ruby
+   Mongoid::Slug.config do |c|
+     #create a block that takes the current object as an argument
+     #and returns the slug.
+     c.slug = proc{|cur_obj|
+       cur_object.slug_builder.to_url
+     }
+   end
+   ```
+
+2. Define it on model level:
+
+   ```ruby
+   class Caption
+     include Mongoid::Document
+     include Mongoid::Slug
+
+     #create a block that takes the current object as an argument
+     #and returns the slug.
+     slug do |cur_object|
+       cur_object.slug_builder.to_url
+     end
+   end
+   ```
 You can call stringex `to_url` method.
 
+The block defined in the model is given preference over the block defined globally.
```

Comment:
I really think that both of these ways are the same way, just at different levels. So I would say "you can define a slug builder globally and/or override it per model". That also should make clear what happens when you have a global builder and override it for a given model.